caluma/caluma_form/structure.py

```diff
@@ -124,11 +130,25 @@ def children(self):
         ]
 
 
+class MetaObject(Element):
```

Comment:
You could probably inherit from `Element, dict` so you don't have to proxy through to the "inner" dict.

Also, seeing as it doesn't actually do much - is the `Element` inheritance really required? Could you do with using a simple dict instead?